sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/JmsIOTest.java

```diff
@@ -333,6 +335,13 @@ public void testCheckpointMark() throws Exception {
     assertEquals(0, count(QUEUE));
   }
 
+  @Test
+  public void testJmsCheckpointMarkAvroEncoding() {
+    AvroCoder<JmsCheckpointMark> avroCoder = AvroCoder.of(JmsCheckpointMark.class);
+
+    assertNotNull(avroCoder);
```

Comment:
Can you please add a test here that validates that we can encode and decode an instance of  `JmsCheckpointMark` correctly. For example by using: `CoderProperties.coderDecodeEncodeEqual(avroCoder, checkpointMark);` or `CoderProperties.structuralValueDecodeEncodeEqual(avroCoder, checkpointMark);`